src/helpers/index.ts

```diff
@@ -0,0 +1,15 @@
+export const setStyles = <T>(stylesObject: T, el: HTMLElement) => {
```

Comment:
we don't follow this pattern with a "helpers" folder in src. Can you move these to src/Reddit/util.ts?